app/controllers/show-project.js

```diff
@@ -41,7 +41,7 @@ export default class ShowProjectController extends Controller {
   get isUserAssignedToProject() {
     // user is from currentUser Service
     // currentUser Service injected in show-project route on setUpController
-    if (this.session.isAuthenticated) {
+    if (this.user && this.session.isAuthenticated) {
```

Comment:
fair

Reply:
Yah, it was throwing "cannot access undefined" for  `this.user.assignments` for when the CP was computing before `this.user` was available.